src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int bottomBase;
+    private int topBase;
+    private int hight;
+
+    public IsoscelesTrapezoid(int bottomBase, int topBase, int hight, String color) {

```

Comment:
Consider using the `Color` enum instead of a `String` for the color parameter to maintain type safety and consistency with the rest of the code.